scripts/docs.js

```diff
@@ -0,0 +1,25 @@
+var cpdir = require('ncp').ncp;
+var fs = require('fs');
+var request = require('request');
+var unzip = require('unzip');
+
+var BRANCH = process.env.AFRAME_BRANCH || 'master';
+var TARGET = process.env.AFRAME_TARGET || BRANCH;
+var TMP = 'tmp';
+var url = 'https://github.com/aframevr/aframe/archive/' + BRANCH + '.zip';
+
+console.log('Fetching: ' + url);
+request.get({url: url, encoding: null}, function (err, res, body) {
+  if (err) { throw err; }
+
+  if (!fs.existsSync(TMP)){
+    fs.mkdirSync(TMP);
+  }
+
+  fs.writeFileSync('tmp/aframe.zip', body);
```

Comment:
you'll want to do like `require('path').join(__dirname, 'tmp/aframe.zip')` so that this script can be run from inside `scripts/` (or from anywhere, not just from the main repo directory)